CHANGELOG.md

```diff
@@ -2,189 +2,149 @@
 
 All notable changes to this project will be documented in this file.
 
-## [0.2.12] - 2023-04-24
-
+## [unreleased]
```

Comment:
Typo?

Reply:
There wasn't a tag yet, so the changelog generator got confused :)
Should be fixed now.